ballerina/build.gradle

```diff
@@ -39,6 +39,11 @@ def ballerinaLinuxDistributionZip = file("$project.buildDir/distributions/baller
 def ballerinaMacDistributionZip = file("$project.buildDir/distributions/ballerina-macos-${ballerinaVersion}.zip")
 def ballerinaWindowsDistributionZip = file("$project.buildDir/distributions/ballerina-windows-${ballerinaVersion}.zip")
 
+// Read StdLib config JSON
+def stdLibInputFile = new File("${project.rootDir}/ballerina/resources/stdlib-configs.json")
+def stdLibConfigs = new JsonSlurper().parseText(stdLibInputFile.text)
+
+
```

Comment:
@BuddhiWathsala  Do we need this extra line?

Reply:
Fixed in here https://github.com/ballerina-platform/ballerina-distribution/pull/482/commits/fea47d5b74fcc893a1d5d53267638e2fc193a277